### PR TITLE
enhance: Use ts as msgID for request

### PR DESCRIPTION
**Pull request by xiaocai2333**
issue: #37431
1. Only search/query requests with a consistency level other than strong will skip timestamp assignment.
2. `MsgID` holds little significance for search/query, so duplication is acceptable.

### CODE REVIEW

**Comment by xiaocai2333** [comment]
rerun go-sdk

File: internal/proxy/task_scheduler.go [hunk 1]
```diff
@@ -177,18 +177,14 @@ func (queue *baseTaskQueue) Enqueue(t task) error {
 	var id UniqueID
 	if t.CanSkipAllocTimestamp() {
 		ts = tsoutil.ComposeTS(time.Now().UnixMilli(), 0)
-		id, err = globalMetaCache.AllocID(t.TraceCtx())
```
Comment (xiaofan-luan) [inline internal/proxy/task_scheduler.go:180]: should we also remove  globalMetaCache.AllocID?

Comment (xiaocai2333) [reply internal/proxy/task_scheduler.go:180]: yes, allocID is only used here, so I'll clean it up.

**Comment by xiaocai2333** [comment]
rerun go-sdk

**Comment by xiaocai2333** [comment]
rerun go-sdk

**Comment by czs007** [comment]
taskID  is used in map, it's dangerous to use ts as ID.

**Comment by xiaocai2333** [comment]
Perhaps in the future, we could replace ID allocation from rootcoord with a local auto-increment approach.
close it first。
/close

**Pull request closed without merging**